torch/csrc/jit/passes/onnx/function_substitution.cpp

```diff
@@ -2,11 +2,84 @@
 
 #include <torch/csrc/jit/jit_log.h>
 #include <torch/csrc/jit/passes/onnx/helper.h>
+#include <torch/csrc/jit/passes/onnx/naming.h>
 
 namespace torch {
 namespace jit {
 
+namespace {
+
+const std::string top_module_variable_name = "";
+
+std::string TidyClassNameFromTorchScript(
+    const c10::optional<c10::QualifiedName>& class_name) {
+  if (!class_name) {
+    return "UNKNOWN_CLASS";
+  }
+  std::string out = "";
+  for (const auto& atom : class_name->atoms()) {
+    bool isInternalTorchAtom = (atom == "__torch__");
```

Comment:
nit: snake case for var names? https://google.github.io/styleguide/cppguide.html#Variable_Names